src/stepper/Stepper.tsx

```diff
@@ -230,11 +235,14 @@ export default class Stepper extends PureComponent<StepperProps, StepperState> {
             type="button"
             className="kirk-stepper-increment"
             status={ButtonStatus.UNSTYLED}
-            disabled={isMax}
+            disabled={disabled || isMax}
             isBubble
             {...this.createButtonListeners(this.increment)}
           >
-            <PlusIcon iconColor={isMax ? color.lightGray : color.blue} size={buttonSize} />
+            <PlusIcon
+              iconColor={disabled || isMax ? color.lightGray : color.blue}
```

Comment:
Icons all have the "isDisabled" (I know) prop that should be enough to gray them out

Reply:
Can't find the code related to that. Can you point it out?

Reply:
https://github.com/blablacar/ui-library/blob/master/src/_utils/icon/index.tsx#L14

Reply:
Thx ! I'll try to clean this a bit

Reply:
The difference: disabled icon is gray, disabled stepper icon is lightGray 

```
  gray: '#DDD',
  lightGray: '#EDEDED',
```

Is it an issue or can we just put gray everywhere?

Ping @wakooka

Reply:
Now using `isDisabled` from icons